neuro_evolution_ctrnn/tools/result_handler.py

```diff
@@ -51,6 +52,10 @@ def write_result(self, hof, log, time_elapsed: float, individual_size: int, inpu
         with open(os.path.join(self.result_path, 'git.diff'), 'wb') as diff_file:
             diff_file.write(self.git_diff)
 
+        if final_checkpoint_data:
+            with open(os.path.join(self.result_path, 'checkpoint.pkl'), "wb") as cp_file:
+                pickle.dump(final_checkpoint_data, cp_file, protocol=pickle.HIGHEST_PROTOCOL, fix_imports=False)
```

Comment:
Was macht `pickle.HIGHEST_PROTOCOL` bzw. warum nutzt du das?

Reply:
gute frage lol. 

ich bin mir 90% sicher, dass in irgendwelchen fällen der checkpoint am ende nicht wiederhergestellt werden konnte. Wenn Du willst, kann ich etwas in recherchieren und versuchen herauszufinden warum genau das hier gebraucht wird (und ob es überhaupt gebraucht wird)